packages/mandelbrot/package.json

```diff
@@ -52,6 +51,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
+    "@frctl/web": "^0.0.0",
```

Comment:
Since peerDependencys prompt the end-user to install them, this would not make a good peerDep - if a user installs `@frctl/mandelbrot`, they should not need `@frctl/web`.

Internal dependencies are regular deps (like the web and core packages). These are the ones users should not need to install to actually use Fractal.

peerDeps are more like "users can combine these packages to make most of Fractal". This is, then, the fractal package, themes, adapters. peerDep version says what should be compatible feature-wise. 

devDeps are deps that are only for development (testing etc) and peerDeps that are necessary for running tests.

(npm dependency system is very complicated)